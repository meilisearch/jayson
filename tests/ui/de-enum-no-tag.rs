use jayson::DeserializeFromValue;

#[derive(DeserializeFromValue)]
#[jayson(error = jayson::Error)]
enum Enum {
    Variant,
}

fn main() {}
