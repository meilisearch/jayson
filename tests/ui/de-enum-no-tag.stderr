error: Externally tagged enums are not supported yet by jayson. Add #[jayson(tag = "some_tag_key")]
 --> tests/ui/de-enum-no-tag.rs:3:10
  |
3 | #[derive(DeserializeFromValue)]
  |          ^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `DeserializeFromValue` (in Nightly builds, run with -Z macro-backtrace for more info)
